packages/grid/x-grid-data-generator/src/asyncWorker.ts

```diff
@@ -0,0 +1,26 @@
+export default function asyncWorker({ work, tasks, done }) {
+  function myNonEssentialWork(deadline) {
+    // If there is a surplus time in the frame, or timeout
+    while ((deadline.timeRemaining() > 0 || deadline.didTimeout) && tasks.current > 0) {
+      work();
+    }
+
+    if (tasks.current > 0) {
+      // @ts-ignore
```

Comment:
why 3 ts-ignore in this file? How about we fix the compile error properly

Reply:
I don't think it matters that requestIdleCallback is correctly typed, but I can install it's corresponding type definition.